index.html

```diff
@@ -5,8 +5,52 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Microverse Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+    <header>
+        <nav id="mobile-navbar">
+            <img src="assets/menu.png" alt="">
+        </nav>
+    </header>
+    <section id="background-image">
+        <h1 >
+            <span id="name-heading">
+                Hey there<br>
+            I'm Abdul Salam
+            </span><br>
+            <span id="job-role">
+                I',m a Software<br>
+                Developer
+            </span>
+            
+        </h1>        
+        <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p>
+       
+          <section id="social-icons">
+            <a href=""></a><img src="assets/icons/twitter.png" alt="">
+            
+            <a href=""></a><img src="assets/icons/linkedin.png" alt="">
+            
+            <a href=""></a><img src="assets/icons/medium.png" alt="">
+            
+            <a href=""></a><img src="assets/icons/github.png" alt="">
+            
+            <a href=""></a><img src="assets/icons/angellist.png" alt="">
+        </section>
```

Comment:
- [ ] Please use a **ul** tag to group the list of icons and **li** per icon. Each icon should have a link (a tag).
Apart from being semantically correct, the anchor tag will let you connect your social networks with the icons, using href.
😄😄😄

- [ ] Kindly ensure that your design will look as close as possible to the design from Figma. In this case, I want you to reduce the width of this section so the spacing between the icons reduces as well 👇👇 (Check the design below)

Your design 👇👇👇

![Screenshot from 2023-06-02 04-39-59](https://github.com/AbdulSalamAbro/portfolio-website/assets/10439283/b31a851b-c333-4c7e-82c3-92deceaca595)

Design from Figma 👇👇👇

![Screenshot from 2023-06-02 04-40-19](https://github.com/AbdulSalamAbro/portfolio-website/assets/10439283/996a05e6-9d90-4fe5-852f-f382ddf0ca4a)